src/app/feed/feed-card/feed-card.component.ts

```diff
@@ -131,31 +135,32 @@ export class FeedCardComponent implements OnInit {
 		}
 	}
 
-	private sanitizeAndEmbedURLs(links) {
-		links.forEach((link, i) => {
-			const videoid = links[i].match(/(?:https?:\/{2})?(?:w{3}\.)?youtu(?:be)?\.(?:com|be)(?:\/watch\?v=|\/)([^\s&]+)/);
+	private sanitizeVideoURLs() {
+		this.feedItem.videos.forEach((video) => {
+			const videoid = video.match(/(?:https?:\/{2})?(?:w{3}\.)?youtu(?:be)?\.(?:com|be)(?:\/watch\?v=|\/)([^\s&]+)/);
```

Comment:
you can also use twitter videos that are extracted. see this -> 
http://loklak.org/api/search.json?q=AnyQuery&filter=video
```

    {
.
.
.
.
.
    "videos": [
        "https://video.twimg.com/ext_tw_video/877218026457894913/pu/vid/480x480/AGYBvF98jPR42vB_.mp4",
        "https://video.twimg.com/ext_tw_video/877218026457894913/pu/pl/LHfROVYC5QpWJNwI.m3u8",
        "https://video.twimg.com/ext_tw_video/877218026457894913/pu/vid/720x720/_WQJWOUfya7eMfe2.mp4",
        "https://video.twimg.com/ext_tw_video/877218026457894913/pu/vid/240x240/NLGKdm_ubGAY9uWv.mp4"
      ],
      "videos_count": 4,
      "mentions": [],
      .
.
.
.}
```